akka-bbb-apps/src/universal/conf/application.conf

```
@@ -27,6 +27,11 @@ akka {
     }
 }
 
+client {
+    clientConfigurationFilePath = "/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml"
+    clientConfigurationOverrideFilePath = "/etc/bigbluebutton/bbb-html5.yml"
+}
```

```suggestion
client {
    clientConfigFilePath = "/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml"
    clientConfigOverrideFilePath = "/etc/bigbluebutton/bbb-html5.yml"
}
```

Config is better to keep the name shorter.